package.json

```diff
@@ -22,10 +22,11 @@
     "tsdx": "https://github.com/ibi-group/tsdx"
   },
   "dependencies": {
+    "@babel/runtime": "^7.16.7",
```

Comment:
Is this needed?

Reply:
Yes the deployment failed without this explicit dependency. It looks like it used to be included via some dependency's `yarn.lock` but that seems to have changed.